internal/rules/aws/ec2/require_vpc_flow_logs_for_all_vpcs.go

```diff
@@ -0,0 +1,39 @@
+package ec2
+
+import (
+	"github.com/aquasecurity/defsec/internal/rules"
+	"github.com/aquasecurity/defsec/pkg/providers"
+	"github.com/aquasecurity/defsec/pkg/scan"
+	"github.com/aquasecurity/defsec/pkg/severity"
+	"github.com/aquasecurity/defsec/pkg/state"
+)
+
+var CheckRequireVPCFlowLogs = rules.Register(
+	scan.Rule{
+		AVDID:       "AVD-AWS-0164",
+		Aliases:     []string{"aws-autoscaling-enable-at-rest-encryption"},
+		Provider:    providers.AWSProvider,
+		Service:     "ec2",
+		ShortCode:   "require-vpc-flow-logs-for-all-vpcs",
+		Summary:     `VPC Flow Logs is a feature that enables you to capture information about the IP traffic going to and from network interfaces in your VPC. After you've created a flow log, you can view and retrieve its data in Amazon CloudWatch Logs. It is recommended that VPC Flow Logs be enabled for packet "Rejects" for VPCs.`,
```

Comment:
The documentation from the issue get added here

This is what is found in the avd_docs